subworkflows/nf-core/fasta_newick_epang_gappa/meta.yml

```diff
@@ -0,0 +1,96 @@
+name: "fasta_newick_epang_gappa"
+description: Run phylogenetic placement with a number of query sequences plus a reference alignment and phylogeny. Used in nf-core/phyloplace.
+keywords:
+  - phylogenetic placement
+  - phylogenetics
+  - alignment
+  - fasta
+  - newick
+modules:
+  - hmmer/hmmbuild
+  - hmmer/hmmalign
+  - hmmer/eslalimask
+  - hmmer/eslreformat
+  - mafft
+  - epang/place
+  - epang/split
+  - gappa/examinegraft
+  - gappa/examineassign
+  - gappa/examineheattree
+input:
+  - ch_pp_data:
+    description: |
+      Structure: [
+        meta: val(meta),
+        data: [
+          alignmethod:  'hmmer',
+          queryseqfile: file("https://github.com/nf-core/test-datasets/raw/phyloplace/testdata/PF14720_3_sequences.faa", checkIfExists: true),
+          refseqfile:   file("https://github.com/nf-core/test-datasets/raw/phyloplace/testdata/PF14720_seed.alnfaa", checkIfExists: true),
+          refphylogeny: file("https://github.com/nf-core/test-datasets/raw/phyloplace/testdata/PF14720_seed.ft.LGCAT.newick", checkIfExists: true),
+          model:        "LG",
+          taxonomy:     file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/gappa/gappa_taxonomy.tsv", checkIfExists: true)
```

Comment:
```suggestion
          queryseqfile: path("*.faa"),
          refseqfile:   path("*.alnfaa"),
          refphylogeny: path("*.newick"),
          model:        "LG",
          taxonomy:     path("*.tsv")
```

Reply:
Sorry for this stupid mistake -- it was late yesterday! :-S